src/ds5/ds5-device.h

```diff
@@ -107,6 +108,11 @@ namespace librealsense
         lazy<std::vector<uint8_t>> _color_calib_table_raw;
         std::shared_ptr<lazy<rs2_extrinsics>> _color_extrinsic;
         bool _is_locked = true;
+
+        std::shared_ptr<limits_option>  _gain_limit_toggle_control;
```

Comment:
`_gain_limit_toggle_control`  and  `_ae_limit_toggle_control` are registered directly into the options container. No need for extra owner